subsys/net/lib/zperf/zperf_shell.c

```diff
@@ -1230,7 +1231,8 @@ void zperf_shell_init(void)
 		ret = net_addr_pton(AF_INET, DST_IP4ADDR,
 				    &in4_addr_dst.sin_addr);
 		if (ret < 0) {
-			NET_WARN("Unable to set IP %s",
+			NET_WARN("Unable to set destination %s address %s",
+				 "IPv4",
 				  DST_IP4ADDR ? DST_IP4ADDR
 					      : "(Default IPv4 destination address not set)");
```

Comment:
Perhaps we could just say here `(Default destination address not set)` so that the string could be shared between IPv4 and IPv6 print. The "IPv4" is already printed just before this one so it is obvious what IP address we are talking about.
Same comment to IPv6 print.

Reply:
`(not set)` this should suffice as I had added `destination` to the main print itself.